dallinger/experiments/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""A home for all Dallinger Experiments. Experiments should be registered
+with a ``setuptools`` ``entry_point`` for the ``dallinger.experiments`` group."""
```

Comment:
We should write actual documentation about this, not just a comment in the code.